data-service/src/main/java/uk/gov/gchq/palisade/service/data/web/DataController.java

```diff
@@ -53,10 +67,24 @@ public DataController(final DataService service) {
     @PostMapping(value = "/addSerialiser", consumes = "application/json", produces = "application/json")
```

Comment:
given we are now using pre population, I don't think we need to have a public rest end point for adding the serialisers, which means we can remove the addSerialiserRequest class. I would leave the data service interface addSerialiser method though

Reply:
The AddSerialiserRequest has been deleted and the rest end point has been removed.

Reply:
These have been removed, required some changes to the integration tests.